partials/logo.php

```diff
@@ -70,16 +70,23 @@
 	} else { // Organisational style display.
 		?>
 		<div class="nhsuk-header__logo">
-			<a href="<?php echo esc_url_raw( get_home_url() ); ?>" class="nhsuk-header__link" aria-label="<?php bloginfo( 'name' ); ?> homepage">
-				<span class="nhsuk-header__link">
-					<?php
-					get_template_part( 'partials/nhslogo' );
+            <a href="<?php echo esc_url_raw(get_home_url()); ?>" class="nhsuk-header__link" aria-label="<?php bloginfo('name'); ?> homepage">
+				<?php
+				get_template_part('partials/nhslogo');
+				$logo_lines = split_text($logo_line_1);
+				?>
+				<span class="nhsuk-organisation-name">
+				<?php echo esc_html($logo_lines['line_1']);
```

Comment:
Having the standard index array change in place we can reformat this bit of code:
```
<?php
foreach ($logo_lines as $id => $logo_line) :
    if ($id==0) :
        ?>
        <span class="nhsuk-organisation-name"><?php echo esc_html($logo_line); ?></span>
        <?php
    else :
        ?>
        <span class="nhsuk-organisation-name-split"><?php echo esc_html($logo_line); ?></span>
        <?php
    endif;
endforeach;
?>
<span class="nhsuk-organisation-descriptor"><?php echo esc_html( $logo_line_2 ); ?></span>
					
```